build/media_source/com_templates/vue/components/settings/editgrid.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+    <div>
+        <!-- Settings for editing grids -->
+        <fieldset>
+            <legend>{{ translate('COM_TEMPLATES_EDIT_GRID') }}</legend>
+            <div class="form-group">
+                <label for="grid_class">{{ translate('COM_TEMPLATES_ADD_CLASS') }}</label>
+                <input id="grid_class" name="grid_class" type="text" v-model="grid_class">
+            </div>
+
+            <div class="btn-group">
+                <button type="button" class="btn btn-success" @click="editGrid">{{ translate('COM_TEMPLATES_SAVE') }}</button>
+                <button type="button" class="btn btn-danger" @click="$emit('reset')">{{ translate('JTOOLBAR_BACK') }}</button>
+            </div>
+        </fieldset>
+    </div>
+</template>
+
+<script>
+export default {
+    name: 'edit-grid',
+    props: {
+        grid: {}
+    },
+    data() {
+        return {
+            grid_class: this.grid.options.class,
+        }
+    },
+    methods: {
+        editGrid() {
+            if (this.grid_class !== '') {
+                this.grid.options.class = this.grid_class;
```

Comment:
Why we are updating props in child component ?  You are taking advantage of the weakness of Vue. ;P   

This is a very bad practice. This code gets breaks if something change in Vue and its also breaks your pagebuilder.

Read about one way data binding. https://vuejs.org/v2/guide/components-props.html#One-Way-Data-Flow